lib/hpax.ex

```diff
@@ -43,9 +43,10 @@ defmodule HPAX do
       encoding_context = HPAX.new(4096)
 
   """
-  @spec new(non_neg_integer()) :: Table.t()
-  def new(max_table_size) when is_integer(max_table_size) and max_table_size >= 0 do
-    Table.new(max_table_size)
+  @spec new(non_neg_integer(), HPAX.Table.huffman_strategy()) :: Table.t()
+  def new(max_table_size, huffman \\ :always)
```

Comment:
Default to true

Reply:
If we want to be future proof, maybe we can have `options` here rather than a single `huffman` value?

```elixir
def new(max_table_size, options \\ []) do
  huffman = Keyword.get(options, :huffman, :always)
end
```